README.md

```diff
@@ -21,6 +21,8 @@ Options:
   --odoo-version TEXT           Project's Odoo version (e.g. 16.0)  [required]
   --project-name TEXT           Name of the project, will be the name of
                                 category of local addons (default: Local)
+  --distinct-oca                Add disctinction for third party addons coming
+                                from OCA
```

Comment:
We'll need a `--reset-cache` to force a cache purge.

Reply:
Or --refresh-cache to update cached entries.

Reply:
I prefer the option `--reset-cache` to clear cache before execution